README.md

```diff
@@ -8,110 +8,88 @@
 computing environment for tenant's zero-trust use case.
 
 This project is designed to provide cloud native measurement for the full measurement
 chain from TEE TCB -> Firmware TCB -> Guest OS TCB -> Cloud Native TCB as follows:
 
 ![](/docs/cc-full-meaurement-chain.png)
 
-_NOTE: Different with traditional trusted computing on non-confidential environment,
+_NOTE: Different from traditional trusted computing on non-confidential environment,
 the measurement chain is not only started with Guest's `SRTM` (Static Root Of Measurement)
-but also need include the TEE TCB, because the CC VM environment is created by TEE
-via `DRTM` (Dynamic Root of Measurement) like Intel TXT on the host._
+but it also needs to include the TEE TCB, because the CC VM environment is created by TEE
+via `DRTM` (Dynamic Root of Measurement) like Intel® TXT on the host._
 
-From the perspective of tenant's workload, `CCNP` will expose the [CC Trusted API](https://github.com/cc-api/cc-trusted-api)
-as the unified interfaces across diverse trusted foundations like `RTMR+TDMR+CCEL`
-and `PCR+TPM2`. The definitions and structures follows standard specifications like
-[TCG PC Client Platform TPM Profile Specification](https://trustedcomputinggroup.org/resource/pc-client-platform-tpm-profile-ptp-specification/),
-[TCG PC Client Platform Firmware Profile Specification](https://trustedcomputinggroup.org/resource/pc-client-specific-platform-firmware-profile-specification/)
+From the perspective of a tenant's workload, `CCNP` will expose the [CC Trusted API](https://github.com/cc-api/cc-trusted-api)
+as the unified interfaces across diverse trusted foundations like `RTMR+MRTD+CCEL`
+and `PCR+TPM2`. Learn more details of CCNP design at [CCNP documentation](https://intel.github.io/confidential-cloud-native-primitives/).
 
 ![](/docs/ccnp-architecture-high-level.png)
 
-This project should also be able deployed on [diverse cloud native PaaS frameworks](https://www.redhat.com/en/blog/confidential-computing-use-cases) like confidential cluster, container, `kubevirt` etc.
-An example of landing architecture on confidential cluster is as follows, please
-refer [detail deployment steps](/deployment/README.md)
-
-![](/docs/ccnp-landing-confidential-cluster.png)
-
-Finally, the full trusted chain will be measured into CC report as follows using
-TDX as example:
+Finally, the full trusted chain will be measured into a CC report as follows using Intel TDX as an example:
 
 ![](/docs/cc-full-measurement-tdreport.png)
 
 _NOTE:_
 
 - The measurement of TEE, Guest's boot, OS is per CC VM, but cluster/container measurement
 might be per cluster/namespace/container for cloud native architecture.
-- Please refer structure [`TDREPORT`](https://github.com/tianocore/edk2/blob/master/MdePkg/Include/IndustryStandard/Tdx.h)
-
-
-## 2. Design
+- Please refer to structure [`TDREPORT`](https://github.com/tianocore/edk2/blob/master/MdePkg/Include/IndustryStandard/Tdx.h)
```

Comment:
The other two bullet points ended with a period  .   so for consistency I would add a period here too.

Reply:
Good catch.